Fastfile

```diff
@@ -92,7 +92,7 @@ def _build_and_deploy_to_hockey
 end
 
 def _build_number
-  use_timestamp = ENV['TAB_USE_TIME_FOR_BUILD_NUMBER'] || false
+  use_timestamp = ENV['TAB_USE_TIME_FOR_BUILD_NUMBER'].to_s == 'true' || false
```

Comment:
Would this work in the scenario where the value is `'TRUE'`?
Is that a valid scenario? 
If it is, maybe we could lowercase the value just in case before evaluating the expression.

Reply:
I guess it is! Easy to add

Reply:
Updated